package com.nt.exception;

public class TestThrowThrows2 {
	public static void main(String[] args){
		try{
			m1();
		}
	catch(Exception e){
		
	}
	static void m1() throws RuntimeException{
		
	}
}

